pycbc/conversions.py

```diff
@@ -446,30 +500,62 @@ def remnant_mass_from_mass1_mass2_spherical_spin_eos(
         The tilt angle of the spin of mass1. Default = 0 (aligned w L).
     eos : str, optional
         Name of the equation of state being adopted. Default is '2H'.
+    spin2a : float, optional
+        The dimensionless magnitude of the spin of mass2. Default = 0.
+    spin2pol : float, optional
+        The tilt angle of the spin of mass2. Default = 0 (aligned w L).
+    swap_companions : boolean, optional
+        If mass2 > mass1, swap mass and spin of object 1 and 2 prior
+        to applying the fitting formula (otherwise fail). Default is False.
+    ns_bh_mass_boundary : float, optional
+        If mass2 is greater than this value, the NS is effectively
+        treated as a BH and the returned value is 0. Default is None.
```

Comment:
Suggest to the user to open the eos file for the EOS they chose and take the maximum NS mass from there.